File: 30DaysOfCode/day01_dataTypes/dataTypes.php
<?php
$handle = fopen ("php://stdin","r");
$i = 4;
$d = 4.0;
$s = "HackerRank ";

$another_i = fgets($handle);
$another_d = fgets($handle);
$another_s = fgets($handle);

echo $i + $another_i . "\n";
echo number_format($d + $another_d, 1) . "\n";
echo $s . $another_s;

fclose($handle);
?>

File: 30DaysOfCode/day01_dataTypes/dataTypes.pl
$i = 4;
$d = 4.0;
$s = 'HackerRank ';

my $another_i = <STDIN>;
my $another_d = <STDIN>;
my $another_s = <STDIN>;

print $i + $another_i, "\n";
print sprintf("%.1f", $d + $another_d), "\n";
print $s, $another_s;
